control-station/src/components/ControlPanel/style.css

```diff
@@ -0,0 +1,30 @@
+#controlpanel{
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    background-color: black;
+    height: 9%;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.button{
+    height: 60px;
+    width: 200px;
```

Comment:
Issue: Using an explicit `height` and `width` results in content spilling over on shorter views and is not entirely flexible depending on the inner content. Consider using `padding` instead of explicit dimensions.